tests/smoketest/minimal-collectd.conf.template

```diff
@@ -11,7 +11,7 @@ LoadPlugin cpu
 LoadPlugin amqp1
 <Plugin "amqp1">
   <Transport "name">
-    Host "qdr-white.sa-telemetry.svc.cluster.local"
+    Host "saf-default-interconnect.sa-telemetry.svc.cluster.local"
```

Comment:
:+1: Just noting that you can shorten this to "saf-default-interconnect"; since containers all have the rest of the name in their DNS search patch (injected by openshift). We don't currently check the hostname on the cert anyways, but currently it shows `subject=/O=cert-manager/CN=saf-default-interconnect` so shouldn't be a problem either way.

Reply:
Neat, you've mentioned this to Paul, but I forgot at the time.